test/cli.spec.js

```diff
@@ -32,15 +42,54 @@ describe("CLI", function(){
     });
 
     it("should succeed with a valid SVG", async function(){
-        const validSvg = "./test/svgs/attr.test.svg";
-        const { failed } = await execCliWith([validSvg]);
+        const { failed } = await execCliWith([VALID_SVG]);
         expect(failed).toBeFalsy();
     });
 
     it("should fail with an invalid SVG", async function(){
-        const invalidSvg = "./test/svgs/elm.test.svg";
-        const { failed, exitCode } = await execCliWith([invalidSvg]);
+        const { failed, exitCode } = await execCliWith([INVALID_SVG]);
+        expect(failed).toBeTruthy();
+        expect(exitCode).toBe(1);
+    });
+});
+
+describe("Configuration", function() {
+    it("should fail passing an unexistent file path to --config", async function() {
+        const { failed, exitCode } = await execCliWith(
+            [VALID_SVG, "--config", "./this/file/does/not-exist.js"],
+        );
         expect(failed).toBeTruthy();
         expect(exitCode).toBe(1);
     });
+
+    it("should succeed passing an existent file path to --config", async function() {
+        const { failed } = await execCliWith(
+            [VALID_SVG, "--config", "test/projects/esm/foo/custom-svglint-config.js"]);
```

Comment:
For consistent styling with the previous test:

```suggestion
        const { failed } = await execCliWith(
            [VALID_SVG, "--config", "test/projects/esm/foo/custom-svglint-config.js"]
        );
```